src/plugins/backlight.ts

```diff
@@ -0,0 +1,43 @@
+import { Plugin, Cordova } from './plugin';
+
+/**
+ * @name Backlight
```

Comment:
Could you please add `@beta` tag above this line? The plugin seems very new and the performance might not be guaranteed.